src/Component/Calculator.js

```diff
@@ -37,39 +35,38 @@ class Calculator extends React.Component {
         </div>
         <div className="calculator d-flex flex-column">
           <div className="calc-row d-flex align-content-stretch">
-            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, 'AC')); }}>AC</button>
-            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '+/-')); }}>+/-</button>
-            <button className="p-5" onClick={() => { this.eventHandler(calculate(this.state, '%')); }}>%</button>
-            <button className="p-5 orange" onClick={() => { this.eventHandler(calculate(this.state, '÷')); }}>÷
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, 'AC')); }}>AC</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '+/-')); }}>+/-</button>
+            <button className="p-5" onClick={() => { eventHandler(calculate(obj, '%')); }}>%</button>
```

Comment:
- [ ] Please work on the functionality of the `%` currently its not working

Reply:
Hi! The functionality works just perfectly from here. Can I know the error you having there?